### PR TITLE
Added default value in variable.tf

### PR DESCRIPTION
**Pull request by sharadkesarwani**
Added default value in variable.tf to avoid asking value for option field value in case of terraform apply.

# Jira Ticket
Ref: https://spotinst.atlassian.net/browse/SPOTAUT-16289

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -61,18 +65,22 @@ variable "health_grace_period" {
...
   description = "An Array holding Availability Zones, this configures the availability zones the Ocean may launch instances in per VNG."
 }
 variable "max_pods_per_node" {
   type        = number
+  default     = null
   description = "The maximum number of pods per node in an AKS cluster."
 }
 variable "enable_node_public_ip" {
   type        = bool
+  default     = false
```
Comment (chandra1-n) [inline variables.tf]: change it to null for bool type also

File: variables.tf [hunk 2]
```diff
@@ -122,49 +133,61 @@ variable "labels" {
 }
 variable "scheduling_shutdown_hours_time_windows" {
   type        = list(string)
+  default     = []
   description = "The times that the shutdown hours will apply."
 }
 variable "scheduling_shutdown_hours_is_enabled" {
   type        = bool
+  default     = false
```
Comment (chandra1-n) [inline variables.tf]: change it to null for bool type also

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix review comments

File: variables.tf (hunk 1 above)
Comment (sharadkesarwani) [reply variables.tf]: Gave null as default value

File: variables.tf (hunk 2 above)
Comment (sharadkesarwani) [reply variables.tf]: Gave null as default value

**Review by chandra1-n — APPROVED** [review]

**Pull request merged**